cmd/oci-create-layer/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/opencontainers/image-tools/image"
+	"github.com/spf13/cobra"
+)
+
+type layerCmd struct {
+	stdout *log.Logger
+	stderr *log.Logger
+}
+
+func main() {
+	stdout := log.New(os.Stdout, "", 0)
+	stderr := log.New(os.Stderr, "", 0)
+
+	cmd := newLayerCmd(stdout, stderr)
+	if err := cmd.Execute(); err != nil {
+		stderr.Println(err)
+		os.Exit(1)
+	}
+}
+
+func newLayerCmd(stdout, stderr *log.Logger) *cobra.Command {
+	v := &layerCmd{
+		stdout: stdout,
+		stderr: stderr,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "create-layer [child] [parent]",
```

Comment:
This should probably write the layer tar to stdout unless the user sets an option to write to a particular filename.